examples/nlp/dialogue_state_tracking/data/multiwoz/schema.json

```diff
@@ -0,0 +1,636 @@
+          "hotel-internet": "dontcare",
+          "hotel-name": "dontcare",
+          "hotel-area": "dontcare"
+        }
+      }
+    ]
+  },
+  {
+    "service_name": "train",
+    "slots": [
+      {
+        "name": "train-destination",
+        "description": "the city you want to go to",
+        "possible_values": [
+          "birmingham new street",
+          "bishops stortford",
+          "bournemouth",
+          "broxbourne",
+          "cambridge",
+          "centre",
+          "city centre north",
+          "copper kettle",
+          "curry prince",
+          "ely",
+          "glastonbury",
+          "gourmet burger kitchen",
+          "huntingdon marriott hotel",
+          "huntington marriott",
+          "kings lynn",
+          "leicester",
+          "liverpool",
+          "liverpool street",
+          "london",
+          "london kings cross",
+          "london liverpool street",
+          "norway",
+          "norwich",
+          "peterborough",
+          "stansted airport",
+          "stevenage"
+        ],
+        "is_categorical": true
+      },
+      {
+        "name": "train-arriveby",
+        "description": "when should the train reach your destination",
+        "possible_values": [],
+        "is_categorical": false
+      },
+      {
+        "name": "train-departure",
+        "description": "the location where you want to catch the train from",
+        "possible_values": [
+          "alpha milton",
+          "aylesbray lodge guest",
+          "birmingham new street",
+          "bishops stortford",
+          "brookshite",
+          "broxbourne",
+          "cafe uno",
+          "camboats",
+          "cambridge",
+          "cineworld",
+          "city hall",
+          "duxford",
+          "east london",
+          "ely",
+          "hamilton lodge",
+          "huntingdon",
+          "kings lynn",
+          "leicester",
+          "liverpool",
+          "london",
+          "london kings cross",
+          "london liverpool",
+          "london liverpool street",
+          "norwich",
+          "panahar",
+          "peterborough",
+          "stansted airport",
+          "stevenage",
+          "stratford",
+          "wandlebury country park"
+        ],
+        "is_categorical": true
+      },
+      {
+        "name": "train-day",
+        "description": "the day of the journey",
+        "possible_values": [
+          "friday",
+          "monday",
+          "saturday",
+          "sunday",
+          "thursday",
+          "tuesday",
+          "wednesday"
+        ],
+        "is_categorical": true
+      },
+      {
+        "name": "train-bookpeople",
+        "description": "number of tickets to buy",
+        "possible_values": [
+          "1",
+          "10",
+          "15",
+          "2",
+          "3",
+          "4",
+          "5",
+          "6",
+          "7",
+          "8",
+          "9"
+        ],
+        "is_categorical": true
```

Comment:
I guess many of those guys should go into "is_numerical" rather than "is_categorical" ;)